src/Bicep.Core/Semantics/BuiltInNamespaceSymbol.cs

```diff
@@ -1,18 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
+using Bicep.Core.TypeSystem;
 using Bicep.Core.TypeSystem.Types;
 
 namespace Bicep.Core.Semantics
 {
     public class BuiltInNamespaceSymbol : Symbol, INamespaceSymbol
```

Comment:
Not for this PR, but I wonder if it makes sense to rename this to `ImplicitNamespaceSymbol` for sake of consistency.

Reply:
Good idea. Will handle this in a follow-up PR as it'll result in a large set of changes.